docs/install/collecting-metrics/README.md

```diff
@@ -0,0 +1,97 @@
+This document describes how to set up the
+[OpenTelemetry Collector](https://opentelemetry.io/docs/collector/about/) to
+receive metrics from the Knative infrastructure components and distribute them
+to Prometheus. [OpenTelemetry](https://opentelemetry.io/) is a CNCF project to
```

Comment:
We could also use the definition of what is advertised at the [official site](https://opentelemetry.io/):

> OpenTelemetry (CNF project) is an observability framework for cloud native software. It provides a collection of tools, APIs, and SDKs for instrumenting, generating, collecting, and exporting telemetry data (metrics, logs, and traces) for analysis in order to understand software's performance and behavior.

Reply:
Thanks, copied it over.